### PR TITLE
[docs] Working with different formats

### PR DESCRIPTION
**Pull request by stevhliu**
This PR officially adds @patrickvonplaten's post on the forums about converting `.ckpt` files and cleans up the KerasCV section a bit to only focus on conversion.

This also starts the process of documenting all the different formats we work with and how to convert from one to another as discussed [here](https://github.com/huggingface/diffusers/pull/3333#discussion_r1185569162) and [here](https://huggingface.slack.com/archives/C041M7M98JV/p1682979328189359?thread_ts=1682963234.663369&cid=C041M7M98JV). I'm sure I'm missing some, so please feel free to let me know about other formats that should be included!

### CODE REVIEW

File: docs/source/en/using-diffusers/kerascv.mdx [hunk 1]
```diff
@@ -10,170 +10,108 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
 specific language governing permissions and limitations under the License.
 -->
 
-# Using KerasCV Stable Diffusion Checkpoints in Diffusers
+# Load different Stable Diffusion formats
 
-<Tip warning={true}>
+Stable Diffusion models are available in different formats depending on the framework they're trained and saved with and where you download them from. Converting these formats for use in 🤗 Diffusers allows you to use all the features supported by the library, such as [using different schedulers](schedulers) for inference or [building your custom pipeline](write_own_pipeline).
```
Comment (sayakpaul) [inline docs/source/en/using-diffusers/kerascv.mdx:15]: I would maybe also highlight the speed optimization that we offer from the library too as that is pretty convenient for practitioners.

File: docs/source/en/using-diffusers/kerascv.mdx [hunk 2]
```diff
@@ -10,170 +10,108 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
+
+<Tip>
 
-This is an experimental feature.
+We highly recommend using the `.safetensors` format because it is safer than traditional pickled files which are vulnerable and can be exploited to execute any code on your machine (learn more in the [Load safetensors](using_safetensors) guide).
```
Comment (sayakpaul) [inline docs/source/en/using-diffusers/kerascv.mdx:19]: Is the hyperlinking right?

File: docs/source/en/using-diffusers/kerascv.mdx [hunk 3]
```diff
@@ -10,170 +10,108 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
 ```py
 from diffusers import DiffusionPipeline
 
-pipeline = DiffusionPipeline.from_pretrained("sayakpaul/textual-inversion-cat-kerascv_sd_diffusers_pipeline")
-pipeline.to("cuda")
-
-placeholder_token = "<my-funny-cat-token>"
-prompt = f"two {placeholder_token} getting married, photorealistic, high quality"
-image = pipeline(prompt, num_inference_steps=50).images[0]
+pipeline = DiffusionPipeline.from_pretrained("")
```
Comment (sayakpaul) [inline docs/source/en/using-diffusers/kerascv.mdx:104]: Why there's no path here?

File: docs/source/en/using-diffusers/kerascv.mdx [hunk 4]
```diff
@@ -10,170 +10,108 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
+from diffusers import DiffusionPipeline
 
-pipeline = DiffusionPipeline.from_pretrained("sayakpaul/textual-inversion-cat-kerascv_sd_diffusers_pipeline")
-pipeline.scheduler = DPMSolverMultistepScheduler.from_config(pipeline.scheduler.config)
+pipeline = DiffusionPipeline.from_pretrained("")
 pipeline.to("cuda")
 
-placeholder_token = "<my-funny-cat-token>"
-prompt = f"two {placeholder_token} getting married, photorealistic, high quality"
+prompt = ""
```
Comment (sayakpaul) [inline docs/source/en/using-diffusers/kerascv.mdx:115]: Why is this incomplete?

File: docs/source/en/using-diffusers/kerascv.mdx (hunk 2 above)
Comment (stevhliu) [reply docs/source/en/using-diffusers/kerascv.mdx:19]: It works in the preview!

File: docs/source/en/using-diffusers/kerascv.mdx [hunk 5]
```diff
@@ -10,170 +10,108 @@ an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express o
...
+[KerasCV](https://keras.io/keras_cv/) supports training for [Stable Diffusion](https://github.com/keras-team/keras-cv/blob/master/keras_cv/models/stable_diffusion) v1 and v2. However, it offers limited support for experimenting with Stable Diffusion models for inference and deployment whereas 🤗 Diffusers has a more complete set of features for this purpose, such as different [noise schedulers](https://huggingface.co/docs/diffusers/using-diffusers/schedulers), [flash attention](https://huggingface.co/docs/diffusers/optimization/xformers), and [other 
+optimization techniques](https://huggingface.co/docs/diffusers/optimization/fp16).
 
-## Using the Converted Model in Diffusers 
+Stable Diffusion models trained with KerasCV are saved in the `.pb` format, though you may also encounter the older `.h5` format. The [Convert KerasCV](https://huggingface.co/spaces/sayakpaul/convert-kerascv-sd-diffusers) Space converts the `.pb` or `.h5` files to PyTorch, and then wraps them in a [`StableDiffusionPipeline`] so it is ready for inference. The converted checkpoint is stored in a repository on the Hugging Face Hub.
```
[the remark this answers is not in the document]
Comment (stevhliu) [reply docs/source/en/using-diffusers/kerascv.mdx:86]: Ah ok, thanks for the clarification! So a checkpoint like this [one](https://huggingface.co/keras-dreambooth/monkey_island_style/tree/main) won't work then?

**Comment by patrickvonplaten** [comment]
Looks good to me! @sayakpaul is the expert here I believe :-)

File: docs/source/en/using-diffusers/kerascv.mdx (hunk 5 above)
[the remark this answers is not in the document]
Comment (sayakpaul) [reply docs/source/en/using-diffusers/kerascv.mdx]: Yeah they would also work. I meant to say that the original Stable Diffusion implementation in KerasCV ships the weights in h5. 

But in this case, your statements are perfectly fine. Sorry about the confusion.

**Review by sayakpaul — APPROVED** [review]
LGTM <3 

Maybe rename the filename (`docs/source/en/using-diffusers/kerascv.mdx`) more appropriately?
